### ISSUE TITLE
docs: need a react demo for subscription.

### ISSUE THREAD

**Issue by liaodalin19903** [trpc/trpc#6313]
### Area of Improvement

https://github.com/trpc/trpc/tree/next/examples 

### Link to related docs

https://trpc.io/docs/v10/subscriptions

### Additional information

Hello. Excuse me. Could you provide a React demo here instead of one based on Next? 

### 👨‍👧‍👦 Contributing

- [ ] 🙋‍♂️ Yes, I'd be down to file a PR implementing the suggested changes!

**Comment by KATT** [comment]
https://github.com/trpc/examples-next-sse-chat

**Comment by Dewald928** [comment]
I forked and deployed this on vercel. But it doesn't work as expected. 

I assume trpc subscriptions doesn't work when deployed on vercel because of the serverless nature, correct?

**Comment by KATT** [comment]
> I assume trpc subscriptions doesn't work when deployed on vercel because of the serverless nature, correct?

Correct. I'd recommend using Railway or Render due to their simplicity
